cardano-prelude/src/Cardano/Prelude/Safe.hs

```diff
@@ -131,8 +131,8 @@ at_ ys o
   | otherwise = f o ys
   where
     f 0 (x:_) = Right x
-    f i (_:xs) = f (i-1) xs
-    f i [] = Left ("index too large, index=" ++ show o ++ ", length=" ++ show (o-i))
+    f i (_:xs) = f (pred i) xs
+    f i [] = Left ("index too large, index=" ++ show o ++ ", length=" ++ show (pred i))
```

Comment:
`GHC.Num(-)` is not `Safe`, so we can use `GHC.Enum(pred)` instead.

https://github.com/tomjaguarpaw/tilapia/issues/137